### PR TITLE
DSE 6.8.49 Release Notes

### PR DESCRIPTION
**Pull request by bereng**
----
## Release Notes Automation
If you name your pull-request as "Product x.y.z Release ...", after merging the
PR, a GitHub Action will automatically create a product version tag "product-x.y.z".

Supported product names are:
 * DSE
 * OpsCenter
 * Studio
 * Luna Streaming

Version supports 3 sets or 4 sets of digits.

### CODE REVIEW

**Review by roxananeo — APPROVED** [review]
+2 LGTM

File: DSE_6.8_Release_Notes.md [hunk 1]
```diff
@@ -4,6 +4,45 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
...
+* Fixed, observe spark.directJoin and spark.directJoinSizeRatio parameters. (DSP-24258)
+
+## 6.8.49 DSE Security
+* Adds partial support for client and internode connections using TLSv1_3. (DSP-23989)
+
+## 6.8.49 DSE Docker
+* Upgraded JDK 8 and 11 versions in DSE Docker images to `8u402` and `11.0.22` respectively. (DSP-24250)
+
+## 6.8.49 DSE Driver
+* Upgrades `dse-java-driver` to handle newer versions of Guava. (DSP-24191)
```
Comment (szymon-miezal) [inline DSE_6.8_Release_Notes.md:40]:
```suggestion
* Upgraded `dse-java-driver` to handle newer versions of Guava. (DSP-24191)
```
The [release notes' guideline](https://datastax.jira.com/wiki/spaces/EN/pages/3374972929/Release+notes+guideline) says that RN should start with a var in past tense, I think we should fix the remaining ones that do not follow this pattern too.

File: DSE_6.8_Release_Notes.md [hunk 2]
```diff
@@ -4,6 +4,45 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
...
+## 6.8.49 DSE Cassandra
+* Secondary Index: Don't fail queries if one node is not available. (DSP-24163)
+* Fix IllegalStateException when flushing range deletes with TWCS split_during_flush=true. (DSP-21571)
+
+## 6.8.49 DSE Spark
+* Fixed, Spark-sql cast errors handling dates on joins. (DSP-24215)
+* Fixed, observe spark.directJoin and spark.directJoinSizeRatio parameters. (DSP-24258)
+
+## 6.8.49 DSE Security
+* Adds partial support for client and internode connections using TLSv1_3. (DSP-23989)
```
Comment (szymon-miezal) [inline DSE_6.8_Release_Notes.md:34]:
```suggestion
* Added partial support for client and internode connections using TLSv1_3. (DSP-23989)
```

File: DSE_6.8_Release_Notes.md [hunk 3]
```diff
@@ -4,6 +4,45 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
...
+* Apache Tomcat® 8.5.94
+* DSE Java Driver 1.10.0-dse-20240520&ast; (DSE *internal-only* version)
+* Netty 4.1.100.1.dse
+* Spark JobServer 0.8.0.54
+
+**NOTE**: above-listed DSE Java Driver is an _internal-version_ only.
+If you're developing applications, please refer to the [Java Driver documentation](https://docs.datastax.com/en/driver-matrix/doc/java-drivers.html) to choose an appropriate version.
+
+## 6.8.49 DSE Core
+* Prevents a Java driver request timeout for `drain()` operations executed via Management API by setting the request timeout to 0. The request timeout change only affects the internal Java driver used in Management API and is only set to 0 explicitly for the `drain()` operation. (DSP-23994)
```
Comment (szymon-miezal) [inline DSE_6.8_Release_Notes.md:23]:
```suggestion
* Prevented a Java driver request timeout for `drain()` operations executed via Management API by setting the request timeout to 0. The request timeout change only affects the internal Java driver used in Management API and is only set to 0 explicitly for the `drain()` operation. (DSP-23994)
```

File: DSE_6.8_Release_Notes.md [hunk 4]
```diff
@@ -4,6 +4,45 @@ Components that are indicated with an asterisk (&ast;) (if any) are known to be
...
+
+## 6.8.49 DSE Cassandra
+* Secondary Index: Don't fail queries if one node is not available. (DSP-24163)
```
Comment (szymon-miezal) [inline DSE_6.8_Release_Notes.md:26]: I would propose to phrase it a bit differently as currently it's difficult to guess what it does from the user perspective: how about _Introduced a flag that allows skipping querying nodes that do not have ready secondary indexes". This behaviour can be enabled by setting `-Ddse.cassandra.index.check_index_status=true`_? @roxananeo as the ticket author, what do you think?

**Review by szymon-miezal — COMMENTED** [review]
I know it has been merged already but I allowed myself to leave a few suggestions.

**Comment by bereng** [comment]
Thx for the feedback @szymon-miezal next time I have to try to remember to review RN from all tickets.

**Pull request merged**